pkg/monitor/resourcewatch/operator/starter.go

```diff
@@ -37,12 +69,22 @@ func RunOperator(ctx context.Context, controllerCtx *controllercmd.ControllerCon
 		return err
 	}
 
-	repositoryPath := "/repository"
-	if repositoryPathEnv := os.Getenv("REPOSITORY_PATH"); len(repositoryPathEnv) > 0 {
-		repositoryPath = repositoryPathEnv
+	var (
+		configStore storage.ResourceWatchStore
+	)
+	if store == monitorStoreStr {
+		eventsClient, err := kubernetes.NewForConfig(controllerCtx.KubeConfig)
+		if err != nil {
+			return err
+		}
+		configStore, err = storage.NewMonitorStorage(artifactDir, eventsClient)
```

Comment:
Would it make any more sense to launch the startClusterOperatorMonitoring funcs, instead of an actual separate store? This is a very loose question just something to think about. Are there benefits to separate store?

Then again, maybe the startX funcs die off depending on the other question posed here about duplicated intervals.